.editorconfig

```diff
@@ -23,14 +22,14 @@ space_after_anonymous_functions = false
 spaces_in_brackets = false
 
 # No Standard.  Please document a standard if different from .js
-[**.yml, **.html, **.css]
+[{*.yml,*.html,*.css}]
 trim_trailing_whitespace = true
 indent_style = tab
 
 # No standard.  Please document a standard if different from .js
-[**.md]
+[*.md]
 indent_style = tab
 
-# Standard at: 
+# Standard at:
 [Makefile]
```

Comment:
Can you clean this up, we don't have a Makefile. Maybe set some standards for the Dockerfile and Procfile.